wp-content/themes/access/views/single-page.twig

```diff
@@ -12,10 +12,13 @@
   </div>
   {% endif %}
 
-  <article>
+  <article itemtype="https://schema.org/mainEntityOfPage">
     <header class="c-header mt-3 print:mt-0">
       <div>
-        <h1 class="text-color-blue-dark">{{ post.title }}</h1>
+        <h1 itemprop="name" class="c-header__descriptor">{{ post.title }}</h1>
+        <h2 class="c-header__information text-color-grey-mid mb-4" itemtype="https://schema.org/dateModified">
```

Comment:
`itemtype`s should be `itemprop` within the scope of the `WebPage` schema and you can drop the url.

```
itemprop="dateModified">
```